### PR TITLE
2023-02-01 release

### PR DESCRIPTION
**Pull request by aleksanderziel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,67 +25,150 @@ A curated list of Zero Knowledge links, mostly focusing on blockchain.
...
+* [The Evolution of Zero-Knowledge Proofs in Web3](https://medium.com/manta-network/the-evolution-of-zero-knowledge-proofs-in-web3-b92a68c41c04) - description of the ZKP with an example on Manta Network.
+* [ZKP Knowledge Base](https://kb.delendum.xyz/) - knowledge base Wikia by Delendum 
 * [Commitment Schemes and Zero-Knowledge Protocols, January 1998](https://www.researchgate.net/publication/221621428_Commitment_Schemes_and_Zero-Knowledge_Protocols) - academic text by Ivan Damgård from Aarhus University. 
 * [The Complexity of Zero Knowledge by Harvard University, October 2007](https://people.seas.harvard.edu/~salil/research/complexityZK.pdf) - academic text by Salil Vadhan from Harvard University. 
 
-### Tutorials and courses
-* [Zero Knowledge Proofs](https://zk-learning.org/) - course on Spring 2023, made by MOOC
+### Tutorials
 * [Zero Knowledge on Casper](https://odra.dev/blog/casper-zk-risc0) - implementing ZK on Casper using risc0.
+* [ZKSnark tutorial](https://github.com/TheBojda/zksnark-tutorial) - JavaScript tutorial for Zero-Knowledge Proofs using SnarkJS and Circom.
```
Comment (kubaplas) [inline README.md:36]: Is there additional documentation for this? There is only some code without explanation. Not a tutorial imo.

File: README.md [hunk 2]
```diff
@@ -16,67 +25,150 @@ A curated list of Zero Knowledge links, mostly focusing on blockchain.
...
+* [ZK Machine Learning]( https://0xparc.org/blog/zk-mnist) - A tutorial and demo by Horace Pan, Francis Ho, and Henri Palacci.
+* [Anatomy of a STARK](https://aszepieniec.github.io/stark-anatomy/) - six-part tutorial about the STARK proof system by Alan Szepieniec.
+* [Libsnark Tutorial](https://github.com/christianlundkvist/libsnark-tutorial) - basics of the libsnark software library for creating zk-SNARKs.
+* [RollupNC Tutorial](https://github.com/therealyingtong/roll_up_circom_tutorial) - tutorial how to build RollupNC. 
+* [Zero Knowledge Proofs](https://github.com/ConsenSys/zero-knowledge-proofs) - Zero Knowledge Proofs and how they can be implemented in Quorum.
+* [Zero Knowledge Proofs Workshop - Zokrates Tutorial](http://extropy.foundation/workshops/zkp/zokrates.html) and [Hash Pre-Image](https://blog.decentriq.com/proving-hash-pre-image-zksnarks-zokrates/) - Tutorial for Zero Knowledge Proofs by Zokrates.
```
Comment (kubaplas) [inline README.md:42]: It's a tutorial to prepare for a workshop, not very useful...

File: README.md [hunk 3]
```diff
@@ -16,67 +25,150 @@ A curated list of Zero Knowledge links, mostly focusing on blockchain.
...
+* [A Practical Guide To Building Zero Knowledge dApps](https://kndrck.co/posts/practical_guide_build_zk_dapps/) - tutorial by Kendrick Tan.
+* [Build a zkSNARK-based Battleship Game on Bitcoin](https://learn.scrypt.io/en/courses/630b1fe6c26857959e13e160) - tutorial learning how to build a zkSnark app.
+* [Circom and SnarkJS tutorial](https://iden3-docs.readthedocs.io/en/latest/iden3_repos/circom/TUTORIAL.html) - tutorial how to create Zero Knowledge zkSnark circuit.
+
+### Conferences and workshops
+* [Crypto Economics Security Conference](https://cesc.io/) - conference in UC Berkeley, usually in November.
+* [Luxembourg Zero Knowledge Days](https://zklux.eu/) - conference from 2019 in Luxembourg, slides available on the page.
```
Comment (kubaplas) [inline README.md:49]: Slides and videos!

File: README.md [hunk 4]
```diff
@@ -16,67 +25,150 @@ A curated list of Zero Knowledge links, mostly focusing on blockchain.
...
+* [Devcon Bogotá](https://devcon.org/) - Devcon for new Ethereum explorers in Colombia, October 2022.
+
+### Upcoming events
+
+* [The 13th BIU Winter School on cryptography](https://cyber.biu.ac.il/event/the-13th-biu-winter-school-on-cryptography/) - course on February 2023 in Bar-Ilan University, Israel.
+* [Zero Knowledge Proofs](https://zk-learning.org/) - course on Spring 2023, made by MOOC.
+* [WEB3 digital summit](https://web3digitalsummit.com/) - Summit in April 2023 in Verona, Italy
+* [zkSummit](https://www.zksummit.com/) - a summit on Zero Knowledge on April 2023 in Lisbon, Portugal.
+* [European WEB3 summit 2023](https://www.web3eurosummit.eu/) - Summit in May 2023 in Brussels, Belgium
+
```
Comment (kubaplas) [inline README.md:61]:
```suggestion
* [RISC Zero Study Club](https://twitter.com/RiscZero/status/1620058982672117766) - Series of zoom meetings with RISC Zero team explaining all things ZK - from 1st Feb to 29th March 2023
```

File: README.md [hunk 5]
```diff
@@ -16,67 +25,150 @@ A curated list of Zero Knowledge links, mostly focusing on blockchain.
...
-* [Aztec Connect](https://aztec.network/) - Aztec Connect is an open source layer 2 network that aims to bring scalability and privacy to Ethereum.
+* [ZKSpace](https://zkspace.info/) - The ZKSpace platform consists of three main parts: a Layer 2 AMM DEX utilizing ZK-Rollups technology ZKSwap v3, a payment service called ZKSquare, an NFT marketplace called ZKSea.
+* [Aztec Connect](https://aztec.network/) - Aztec Connect is an open-source layer 2 network that aims to bring scalability and privacy to Ethereum.
 * [zkSync](https://zksync.io/) - zkSync is a scaling and privacy engine for Ethereum.
-* [Golem](https://www.golem.network/) - Golem is a global, open-source, decentralized supercomputer that anyone can use. It is made up of the combined computing power of the users' machines, from PCs to entire data centers.
+* [Golem](https://www.golem.network/) - Golem is a global, open-source, decentralized supercomputer that anyone can use. It is made up of the combined computing power of the user's machines, from PCs to entire data centers.
```
Comment (kubaplas) [inline README.md:94]: `users'` - it was correct first time

File: README.md [hunk 6]
```diff
@@ -16,67 +25,150 @@ A curated list of Zero Knowledge links, mostly focusing on blockchain.
...
+* [dYdX](https://whitepaper.dydx.exchange/)
+* [Arbitrum Nova](https://github.com/OffchainLabs/nitro/blob/master/docs/Nitro-whitepaper.pdf)
+* [Loopring](https://whitepaper.io/document/26/loopring-whitepaper)
+
+#### Marketplaces
+* [zkPoD](https://github.com/sec-bit/zkPoD-node) - decentralized platform for data exchange between untrusted parties realizing "Payment on Delivery" without any trusted third party.
+* [Ruby Protocol](https://wiki.ruby.xyz/) - private data management framework for Web 3.0. 
+* [Modulo Zero](https://modulozero.xyz/) - on-chain solution for private data exchange.
+
+##### Whitepapers
```
Comment (kubaplas) [inline README.md:120]: Whitepapers for the second time?

**Review by kubaplas — APPROVED** [review]

File: README.md [hunk 7]
```diff
@@ -16,67 +25,150 @@ A curated list of Zero Knowledge links, mostly focusing on blockchain.
...
 * [A Graduate Course in Applied Cryptography](http://toc.cryptobook.us/book.pdf) - book by Dan Boneh and Victor Shoup from 2020.
 * [The MoonMath Manual to zk-SNARKs](https://leastauthority.com/community-matters/moonmath-manual) - book by Mirco Richter.
+* [Computational Complexity: A Modern Approach](https://theory.cs.princeton.edu/complexity/book.pdf) - book by Sanjeev Arora and Boaz Barak from Princeton University, January 2007.
 
 ### Other awesomes
-
 * [Matter Labs - Awesome zero knowledge proofs (zkp)](https://github.com/matter-labs/awesome-zero-knowledge-proofs)
 * [Ventali - Awesome-zk](https://github.com/ventali/awesome-zk)
 * [sCrypt-Inc - Awesome ZKPs](https://github.com/sCrypt-Inc/awesome-zero-knowledge-proofs)
+* [Zero Knowledge Canon](https://a16zcrypto.com/zero-knowledge-canon/)
```
Comment (zie1ony) [inline README.md:174]: Not an awesome. Should be in articles.

File: README.md (hunk 5 above)
Comment (zie1ony) [reply README.md:94]: How is Golem ZK?

File: README.md (hunk 1 above)
Comment (aleksanderziel) [reply README.md:36]: That's a good point. Fixed.

File: README.md (hunk 5 above)
Comment (aleksanderziel) [reply README.md:94]: https://handbook.golem.network/payments/layer-2-payments 
From my point of understanding, they adopted the scalability of Layer 2 solutions for zk-projects such as zkSync or Polygon.  But maybe I am wrong about this.

Comment (zie1ony) [reply README.md:94]: Exactly. It is using zk platforms, but is not zk itself. Please remove it.

File: README.md (hunk 7 above)
Comment (zie1ony) [reply README.md:174]: Changed my mind. It should be here.

**Pull request merged**